tools/src/commands/GenerateDocsAPIData.ts

```diff
@@ -1,44 +1,48 @@
-import { Command } from '@expo/commander';
+import {Command} from '@expo/commander';
```

Comment:
it's been good before 😉 was it your editor that formatted it that way?

```suggestion
import { Command } from '@expo/commander';
```

Reply:
> it's been good before 😉 was it your editor that formatted it that way?

Apparently yes, unfortunately there is no ESLint/Prettier setup for docs, and force running workspace `lint` in `tools` results in thousands of warnings.